patches/minecraft/net/minecraft/entity/ai/attributes/GlobalEntityTypeAttributes.java.patch

```diff
@@ -0,0 +1,23 @@
++   private static final Map<EntityType<? extends LivingEntity>, AttributeModifierMap> FORGE_ATTRIBUTES = new java.util.HashMap<EntityType<? extends LivingEntity>, AttributeModifierMap>();
++   
++   public static AttributeModifierMap put(EntityType<? extends LivingEntity> type, AttributeModifierMap map) {
++       return FORGE_ATTRIBUTES.put(type, map);
++   }
+ 
+    public static AttributeModifierMap func_233835_a_(EntityType<? extends LivingEntity> p_233835_0_) {
+-      return field_233833_b_.get(p_233835_0_);
++       AttributeModifierMap map = field_233833_b_.get(p_233835_0_); return map != null ? map : FORGE_ATTRIBUTES.get(p_233835_0_);
```

Comment:
There's no need to shove the return into the same line. Patches should be made as simple as possible but not to the extend it hurts readability unnecessarily